bindings/netstandard/ElectionGuard/ElectionGuard.Shared/NativeInterface.cs

```diff
@@ -233,18 +234,44 @@ internal static unsafe class PlaintextBallotSelection
         {
             internal unsafe struct PlaintextBallotSelectionType { };
 
+            internal class PlaintextBallotSelectionHandle
```

Comment:
only classes touched in this PR are updated to this new format in order to keep the changes scoped.